spec/support/form_response_helper.rb

```diff
@@ -58,30 +58,6 @@ def valid_data
         phone_number: "0123456789",
         email: "me@example.com",
       },
-      product_details: [
-        {
-          product_id: SecureRandom.uuid,
-          medical_equipment_type: I18n.t("coronavirus_form.questions.medical_equipment_type.options.number_testing_equipment.label"),
```

Comment:
Same here. We should keep the entry for `medical_equipment_type` but move it to the top level so it matches the schema.